composer.json

```
@@ -27,7 +27,7 @@
     },
     "require": {
         "php": ">=5.4.2",
-        "guzzlehttp/psr7": "^1.0"
+        "guzzlehttp/psr7": "^1.7|^2.0"
```

While a single pipe is supported, a double pipe is recommended as the former is still there for BC reasons.
```suggestion
        "guzzlehttp/psr7": "^2 || ^1.7"
```